config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -22,7 +22,16 @@ periodics:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20200513-7c0e31a-master
         command:
           - "runner.sh"
-          - "./scripts/ci-conformance.sh"
+          - "./scripts/ci-entrypoint.sh"
+        env:
+          - name: FOCUS
+            value: "\\[Conformance\\]|\\[NodeConformance\\]"
+          - name: SKIP
+            value: "\\[Slow\\]|\\[Serial\\]|\\[Flaky\\]"
+          - name: PARALLEL
+            value: "true"
+          - name: USE_CI_ARTIFACTS
```

Comment:
this one shouldn't have `USE_CI_ARTIFACTS` set to true, it's meant to test conformance of CAPZ from a known working k8s version (not to test upstream k8s)

Reply:
we could set K8s version explicitly here though, I think right now it just defaults to 1.17.5

Reply:
I see. I will remove it and let the script use the default value.